modules/openapi-generator/src/main/resources/erlang-server/logic_handler.mustache

```diff
@@ -1,7 +1,14 @@
 -module({{packageName}}_logic_handler).
 
 -export([handle_request/4]).
+{{#authMethods}}
+	{{#isApiKey}}
 -export([authorize_api_key/3]).
+	{{/isApiKey}}
+{{/authMethods}}
```

Comment:
This works if the spec contains 1 API key definitions, If it has more than 1, then it won't work. I'll switch it to `hasApiKeyAuth` in another PR. Thanks for your contribution.